app/models/user.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+from sqlalchemy import (
+    Column, Integer, String, DateTime,
+    UniqueConstraint
+)
+from sqlalchemy.orm import relationship
+from models import db
+
+
+class User(db.Model):
+    """
+    Users
+    """
+    __tablename__ = 'users'
+    __table_args__ = (
+        UniqueConstraint('user_id'),
+        UniqueConstraint('user_uid'),
+        {'mysql_engine': 'InnoDB'}
+    )
+
+    user_id = Column(Integer, primary_key=True, autoincrement=True)
+    user_uid = Column(String(512), nullable=True)
```

Comment:
Why do we need to define both of `user_id` and `user_uid`?

Reply:
`user_id` is used for `application_user_roles` table to relate roles. It is used internally only.
`user_uid` is used for the dashboard users to manage other users. It is got from authentication backends such as LDAP, and displayed in the dashboard.

Reply:
I see.
I think `user_uid` is complicated for me since it is similar character to `user_id`. How about using `auth_id` instead? This is good idea when we will support the other ID authentication service such as OpenID connect. All we need is creating `Authentication` table for outer ID service.

Reply:
fixed